run.java

```diff
@@ -183,15 +182,14 @@ private long getJIRAIssueTypeID(GHIssue issue) throws Exception {
     }
 
     private String lookupIssueWithGithubLink(JiraRestClient restClient, String githubLink) {
-        SearchResult searchResults = restClient.getSearchClient().searchJql("project = " + JIRA_PROJECT_CODE + " AND issuetype in (Bug, 'Feature Request', Task) AND 'Git Pull Request' ~ '" + githubLink + "'").claim();
+        SearchResult searchResults = restClient.getSearchClient().searchJql("project = " + JIRA_PROJECT_CODE + " AND issuetype in (Bug, 'Feature', Task) AND 'Git Pull Request' ~ '" + githubLink + "'").claim();
```

Comment:
This fixes a bug creating duplicate issues: https://issues.redhat.com/issues/?jql=project%20%3D%20QUARKUS%20AND%20text%20~%20%22CompletableFuture%20smart%20dispatch%20support%22%20